### ISSUE TITLE
Multiple accounts on same service?

### ISSUE THREAD

**Issue by davej** [LockerProject/Locker#19]
I have multiple accounts on services like Twitter and Facebook. Is it possible to aggregate multiple accounts from the same service (Connector) within the same instance of Locker?

**Comment by smurthas** [comment]
Yep! Simply install a second instance of that connector and it will take you through the same process. You should be able to use the same app id and secret.

**Comment by temas** [comment]
So this is possible with the newer IDRs in a much cleaner way.  What needs to happen is the UI to know which account is being edited/used and then the auth needs to be made an array in the me.json and other places where it's used.  The IDR account id is a reference into that array.

**Comment by dsample** [comment]
The UI certainly needs a way to add multiple accounts, especially for things like Twitter where it's common for people to have multiple.
